jekyll/_cci2/creating-orbs.md

```diff
@@ -0,0 +1,332 @@
+---
+layout: classic-docs
+title: "Creating Orbs"
+short-title: "Creating Orbs"
+description: "Starting point for Creating CircleCI Orbs"
+categories: [getting-started]
+order: 1
+---
+
+Orbs are reusable packages of CircleCI configuration that you may share across projects, enabling you to create encapsulated, parameterized commands, jobs, and executors that can be used across multiple projects.
+
+This section describes how you can:
+
+* Use an existing orb in your configuration
+* Import an orb
+* Create an inline orb in your `config.yml` file
+* Create and publish an orb in the registry
+
+Orbs are made available for use in a configuration through the orbs key in the top level of your config.yml.
```

Comment:
This line feels slightly out of place, like it should either go at the end of the first paragraph in this section, or under the next heading.

Reply:
got it, fixed